pom.xml

```diff
@@ -478,6 +478,7 @@
           <!-- Excludes integration tests when unit tests are run -->
           <excludes>
             <exclude>**/IT*.java</exclude>
+            <exclude>**/*SmokeTest.java</exclude>
```

Comment:
Update the comment above: "integration tests and smoke tests"

Reply:
Done